src/main/scala/com/soundcloud/periskop/client/ExceptionExporter.scala

```diff
@@ -49,6 +49,7 @@ class ExceptionExporter(exceptionCollector: ExceptionCollector) {
     "aggregation_key" -> aggregate.aggregationKey,
     "total_count" -> aggregate.totalCount,
     "severity" -> Severity.toString(aggregate.severity),
+    "timestamp" -> aggregate.timestamp.format(rfc3339TimeFormat),
```

Comment:
shall we use nix epoch instead? much easier to parse and manipulate, not need to worry about time zones

Reply:
@jcreixell would that would also involve changing how it's parsed in Periskop?